src/js/actions/event/data.js

```diff
@@ -0,0 +1,47 @@
+export const GET_EVENT = 'GET_EVENT';
+export const GET_EVENT_REQUEST = 'GET_EVENT_REQUEST';
+export const GET_EVENT_SUCCESS = 'GET_EVENT_SUCCESS';
+export const GET_EVENT_FAILURE = 'GET_EVENT_FAILURE';
+
+const event1 = {
+  event_id: 1,
+  host_user_id: 1,
+  name: 'Lounge party',
+  description: 'Celebrating life',
+  is_poll: true,
+  is_edited: false,
+  _invitees: [2],
+  _what: ['dancing', 'skydiving'],
+  _where: ['Forest', 'Camping'],
+  _when: ['2017-01-03T00:00:00.000Z', '2017-02-14T00:00:00.000Z']
```

Comment:
This is the DATE FORMAT: 'Date Sun Feb 12 2017 10:46:26 GMT+0000 (GMT)' when you pick the date from the PICKER.

This is the TIME FORMAT: 'Time Thu Jan 12 2017 23:46:26 GMT+0000 (GMT)' when you pick the TIME from the PICKER. 

As you can see we have two different sort of timestamps. I am not sure how we can combine these two into ONE?